src/picknik_dual_ur_arm_config/description/picknik_ur_attachments.xacro

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
It would be really nice if we could reuse some of the newer xacro macros from https://github.com/PickNikRobotics/moveit_studio_ur_ws/pull/223, with any needed modifications due to the prefixes, instead of having to make a new one.